core-libs/setup/ssr/logger/transformers/express-log-transformer.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ExpressServerLogger, ExpressServerLoggerContext } from '../loggers';
+
+export interface ExpressLogTransformer {
+  transform(
+    message: string,
+    context: ExpressServerLoggerContext,
+    logger?: ExpressServerLogger
+  ): [string, ExpressServerLoggerContext];
```

Comment:
instead of tuple return type, what about an object with semantic properties:
{ message: string, context: ExpressServerLoggerContext }

This helps mentally interpreting the result of this function and is self-explanatory